GooglePlacesAutocomplete.js

```diff
@@ -322,16 +497,28 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
         }
       };
 
-      request.open(
-        'GET',
-        `${url}/place/details/json?` +
-          Qs.stringify({
-            key: props.query.key,
-            placeid: rowData.place_id,
-            language: props.query.language,
-            ...props.GooglePlacesDetailsQuery,
-          }),
-      );
+      if (props.isNewPlacesAPI) {
+        request.open(
+          'GET',
+          `${url}/v1/places/${rowData.place_id}?` +
+            Qs.stringify({
+              sessionToken,
+              fields: props.fields,
+            }),
```

Comment:
Testing locally, it seems `key` is required even if `sessionToken` is present or the request will return 403.

Reply:
The other fields of this requests should be passed too - `languageCode` and `regionCode`